sktime/transformations/series/difference.py

```diff
@@ -161,60 +169,70 @@ def _check_inverse_transform_index(self, Z):
 
         return is_contained_by_fitted_z, pad_z_inv
 
-    def _fit(self, Z, X=None):
-        """Logic used by fit method on `Z`.
+    def _fit(self, X, y=None):
+        """
+        Fit transformer to X and y.
+
+        core logic
```

Comment:
Ok im not going to mark anymore of these 'core logic' comments

Reply:
same